lib/constructs/dictionary.js

```diff
@@ -1,6 +1,6 @@
 "use strict";
 
-const conversions = require("webidl-conversions");
+const Types = require("../types.js");
```

Comment:
Oh, I see. I guess good to stay consistent, even though that's a weird style choice from @Sebmaster :)